<launch>
  <!-- Command Line Arguments -->
  <arg name="debug" default="false"/>

  <!-- image undistortion node -->
  <include file="$(find jnmouse_ros_examples)/launch/image_undistortion.launch" />
  
  <!-- stereo depth estimation node -->
  <node pkg="jnmouse_ros_examples" name="jnm_depth_estimator" type="stereo_depth_estimation.py" required="true" output="screen">
    <param name="debug" value="$(arg debug)"/>
  </node>
</launch>
